src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -464,17 +464,7 @@ private static bool GetTls13Support()
             return false;
         }
 
-        private static bool GetIsRunningOnMonoInterpreter()
-        {
-#if NETCOREAPP
-            if (IsBrowser)
-                return RuntimeFeature.IsDynamicCodeSupported;
-#endif
-            // This is a temporary solution because mono does not support interpreter detection
-            // within the runtime.
-            var val = Environment.GetEnvironmentVariable("MONO_ENV_OPTIONS");
-            return (val != null && val.Contains("--interpreter"));
```

Comment:
Can we also remove it from `eng/pipelines/coreclr/templates/run-performance-job.yml`?

Reply:
I don't think so.  We still need to _set_ it when we're running on desktop platforms using Mono, since in that case we're just using the normal libraries test host.

The issue is just that it's not sufficient to check for the environment variable setting to detect whether we're using the interpreter.  The RuntimeFeature checks should be accurate on all configurations, AFAIK.